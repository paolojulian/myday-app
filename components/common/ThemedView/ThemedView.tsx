import { View, ViewProps } from 'react-native';

type ThemedViewProps = ViewProps;

export default function ThemedView(props: ThemedViewProps) {
  return <View {...props}></View>;
}
