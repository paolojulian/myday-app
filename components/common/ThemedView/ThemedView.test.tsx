import ThemedView from '@/components/common/ThemedView/ThemedView';
import { render } from '@testing-library/react-native';

describe('TESTING ThemedView Component', () => {
  describe('WHEN the component is rendered', () => {
    it('should match the snapshot', () => {
      const container = render(<ThemedView></ThemedView>);
      expect(container).toMatchSnapshot();
    });
    it('should render without crashing', () => {
      render(<ThemedView>Test</ThemedView>);
      expect(ThemedView).toBeTruthy();
    });
  });
});
