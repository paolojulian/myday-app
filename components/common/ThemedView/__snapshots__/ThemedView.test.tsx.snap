// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TESTING ThemedView Component WHEN the component is rendered should match the snapshot 1`] = `<View />`;
